.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,7 +13,16 @@ Issue #???
 2. Step 2
 3. Step 3
 
-Expected result: ???
+## Key Files to Review
+Main PR-related files
```

Comment:
```suggestion
Group 1
 * File 1
 * File 2
Group 2
 * File 3
 * File 4
```
unless you want people to write paragraphs or comma-separated lists of files

another format is the one I used in #1811, up to you